pyo3-macros-backend/src/pyclass.rs

```diff
@@ -477,7 +477,7 @@ fn impl_class(
     )?;
 
     let (default_class_geitem, default_class_geitem_method) =
-        pyclass_class_geitem(&args.options, &syn::parse_quote!(#cls), ctx)?;
+        pyclass_class_getitem(&args.options, &syn::parse_quote!(#cls), ctx)?;
```

Comment:
If we're fixing the `pyclass_class_getitem` typo, we might as well fix `default_class_geitem` and `default_class_geitem_method` here too (and lower in this function).